src/content/docs/infrastructure/amazon-integrations/connect/aws-multi-account.mdx

```diff
@@ -0,0 +1,112 @@
+
+import infrastructureAwsExploreYourData from 'images/infrastructure_screenshot-crop_aws-explore-data-link.webp'
+
+import infrastructureAwsSeeAccountStatus from 'images/infrastructure_screenshot-crop_aws-see-account-status-dashboard.webp'
+
+import infrastructureAwsAccountStatusDashboard from 'images/infrastructure_screenshot-crop_aws-account-status-dashboard.webp'
+
+
+
+The New Relic UI gives you a complete view of your AWS system, making monitoring much easier. You can view metrics and monitor multiple AWS accounts that send metrics to a single account linked to New Relic. For this feature to work, you must send metrics from different AWS accounts that you want to monitor to a single AWS account. You can then monitor those different AWS accounts with the metrics enriched with the appropriate tags and metadata. You can easily track [performance metrics](/docs/data-apis/understand-data/new-relic-data-types/) while reducing overall infrastructure [costs](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/), which means you'll save time and money.
+
+Having a single view of all your AWS accounts helps you:
+
+* Reduce the number of friction points by setting up a single observe account for AWS.
+
+* Reduce overall infrastructure costs by reducing the number of streams sending metrics to observability solutions.
+
+* Get tagged and metadata-enriched metrics from your AWS accounts.
+
+
+## Before you start [#before-start]
+
+Check the following:
+
+* You've configured [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) to send metrics to a single AWS account that must be connected to New Relic.
+
+* You're using [AWS Resource Groups Tagging API Reference](https://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/overview.html) to collect tags.
+
+* You're using AWS config to collect metadata.
+
+* All IAM roles have access to fetch tags from individual accounts.
+
+* You use the [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) used in the AWS observe account on all AWS accounts to monitor.
+
+
+## Connect your AWS observe account to New Relic [#connect-aws-account]
+
+To start receiving Amazon data with New Relic AWS integrations, connect your AWS account, which receives metrics from all the other AWS accounts you want to monitor, to New Relic.
+
+1. Go to <DNT>**[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure**</DNT>. Under the <DNT>**Integrations**</DNT> section, click <DNT>**AWS**</DNT>. 
+
+2. Click <DNT>**Add AWS account**</DNT>.
+
+    <img
+        title="Add AWS account to New Relic"
+        alt="Add AWS account to New Relic"
+        src={infrastructureAwsAddAccount}
+    />
+
+3. Choose <DNT>**Manually integrate your AWS account**</DNT> intrumentation method.
+
+4. Choose the recommended option, <DNT>**Real-time AWS monitoring (Recommended)**</DNT>.
```

Comment:
Don't think you need "the recommended option" here

Reply:
Done